adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/model/DataImport.java

```diff
@@ -0,0 +1,61 @@
+package org.jvalue.ods.adapterservice.datasource.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+import org.jvalue.ods.adapterservice.datasource.api.rest.v1.Mappings;
+
+import javax.persistence.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+
+@Entity
+@NoArgsConstructor
+@EqualsAndHashCode
+@Getter
+public class DataImport {
+
+  @Id
+  @GeneratedValue
+  private Long id;
+
+  private byte[] data;
+  
+  private Date timestamp;
+
+  @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
That means the datasource is always also retrieved, right? In general, not a bad idea, but why do we not serialize it then? Just a little confused, do we need it?

Reply:
We need its `id` to create the URI to the data in `DataImportMetadata`.

Also the annotations are necessary for Spring to create the database tables and fields properly I think.